### PR TITLE
docs: simplify & update README

### PR DESCRIPTION
**Pull request by gnikit**
This PR proposes a new simpler README for fpm, which redirects a lot of the reading to the fpm website.
Linked to fpm-docs PR: https://github.com/fortran-lang/fpm-docs/pull/87

- [x] Added multiple links to the FPM webpage, homepage, downloading instructions and Getting started and tutorials
- [x] Removed most of the install instructions and redirected to website
- [x] Removed bootstrapping instructions
- [x] Added fpm logo with hyperlink in README
- [x] Fixed link to Manifest in README
- [x] ~~Move bootstrapping to contributing~~ no need
- [x] ~~Mention the Environmental variables in fpm-docs~~ write a tutorial

Fixes Link to documentation in README #776

### CODE REVIEW

**Review by milancurcic — CHANGES_REQUESTED** [review]
If you remove some install instructions but keep others, then the ones you keep implicitly become the recommended ones. Do we really recommend Conda as the best way to get fpm? That requires having and using yet another tool. I think it's okay to keep the Conda and Homebrew instructions, but at the very least also keep and recommend the link to the binary release downloads, as that's the easiest way to get fpm without assuming a 3rd party package manager.

Other changes look good!

**Comment by gnikit** [comment]
@milancurcic Thanks for the quick turnaround in this PR. I agree I was actually meant to include them first but I somehow forgot. I think we might need to do some edits on the fpm-docs repo as well. Specifically, I would suggest:

- moving "Installing fpm" (https://fpm.fortran-lang.org/en/how-to/installation.html) at the very top the TOC on the left and outside "How To"
- Remove the "Design Documents" card from the homepage and add an "Install fpm" card. The order of the cards being from left to right, top to bottom (1. "Install fpm", 2. "Tutorials", 3. "How-To Guides" (now empty), 4. "References")

**Comment by gnikit** [comment]
Please CC anyone else that should review this. Remember to check accompanying PR https://github.com/fortran-lang/fpm-docs/pull/87

**Review by milancurcic — APPROVED** [review]
I think it's good to go, thanks!

**Review by LKedward — APPROVED** [review]
Thanks @gnikit, this is a really nice improvement!

File: README.md [hunk 1]
```diff
@@ -117,15 +86,11 @@ Binary distributions are available for MacOS 11 (Catalina) and 12 (Big Sur) for
 Fpm should be available and functional after those steps.
 For more details checkout the tap [here](https://github.com/fortran-lang/homebrew-fortran).
 
-#### Github Actions
-
-To setup *fpm* within Github actions for automated testing, you can use the [fortran-lang/setup-fpm](https://github.com/marketplace/actions/setup-fpm) action.
+## [Get started](https://fpm.fortran-lang.org/en/tutorial/index.html)
 
-#### Bootstrapping on other platforms
+**Follow our [Quickstart Tutorial](https://fpm.fortran-lang.org/en/tutorial/hello-fpm.html) to get familiar with fpm**.
```
[the remark this answers is not in the document]
Comment (gnikit) [reply README.md:91]: No, just personal taste to point the user to the next step. I can remove it if you want.

**Pull request merged**